src/dotnet/commands/dotnet-new/CSharp_Lib/project.json.pretemplate

```diff
@@ -0,0 +1,12 @@
+{
+  "version": "1.0.0-*",
+  "compilationOptions": {
+    "emitEntryPoint": false
```

Comment:
Do we want to have this in here?  How about just not having compliationOptions at all?  That's how we have most of our .dll project.jsons in the product.